core/src/main/java/org/keycloak/representations/idm/ClientRepresentation.java

```diff
@@ -130,6 +131,14 @@ public void setEnabled(Boolean enabled) {
         this.enabled = enabled;
     }
 
+    public Boolean isTemporaryAdminService() {
+        return temporaryAdminService;
+    }
+
+    public void setTemporaryAdminService(Boolean temporaryAdminService) {
+        this.temporaryAdminService = temporaryAdminService;
+    }
```

Comment:
I believe this should be an [attribute](https://github.com/keycloak/keycloak/blob/6b91ce7074f5897aa0a3b6d265fa5c33cd796b50/core/src/main/java/org/keycloak/representations/idm/ClientRepresentation.java#L62), similarly to users.